zellij-utils/src/envs.rs

```diff
@@ -31,6 +31,11 @@ pub fn get_socket_dir() -> Result<String> {
     Ok(var(SOCKET_DIR_ENV_KEY)?)
 }
 
+pub const ZELLIJ_TMP_LOG_FILE_ENV_KEY: &str = "ZELLIJ_TMP_LOG_FILE";
```

Comment:
note: I was following the architecture of the module and made the const public but I'm not sure why thees keys need to be public.